olive/passes/onnx/genai_model_exporter.py

```diff
@@ -93,7 +94,8 @@ def _run_for_config(
         if not cache_dir:
             cache_dir = os.environ.get("TRANSFORMERS_CACHE", None)
         if not cache_dir:
-            cache_dir = output_model_filepath.parent / "genai_cache_dir"
+            # please do not clean up the cache dir as it may contain model's data
+            os.environ["HF_HOME"] = str(Path(tempfile.gettempdir()) / "hf_cache")
```

Comment:
should this be part of responsibility of user code? genai logic should not change the HF_HOME

Reply:
It should be `cache_dir`, updated.